apps/server/src/modules/files-storage/mapper/file-record.mapper.ts

```diff
@@ -12,9 +12,29 @@ export class FilesStorageMapper {
 		skip?: number,
 		limit?: number
 	): FileRecordListResponse {
-		const responseFileRecords = fileRecords.map((fileRecord) => FilesStorageMapper.mapToFileRecordResponse(fileRecord));
+		const responseFileRecords = fileRecords.map((fileRecord) => FileRecordMapper.mapToFileRecordResponse(fileRecord));
 
 		const response = new FileRecordListResponse(responseFileRecords, total, skip, limit);
 		return response;
 	}
+
+	static mapScanResultParamsToDto(scanResultParams: ScanResultParams): ScanResultDto {
+		const scanResult = new ScanResultDto({
+			status: ScanStatus.VERIFIED,
+			reason: 'Clean',
```

Comment:
Bzg. meines Kommentars oben. Ich denke die reasons die in die DB kommen sind Part der DO Logik, würde mich noch mal dadrin bestärken, das zu verschieben. 

Ich wollte Teile des DO Mapper PRs eh raus trennen, die sich nur darum drehen logic Richtung Entity / DO zu verschieben. Man könnte allerdings auch Code von dort hier rein ziehen.